k9mail/src/main/res/values/strings.xml

```diff
@@ -1280,4 +1280,7 @@ Please submit bug reports, contribute new features and ask questions at
     <string name="openpgp_enabled_error_back">Back</string>
     <string name="openpgp_enabled_error_disable">Disable Encryption</string>
     <string name="openpgp_encryption">OpenPGP Encryption</string>
+    <string name="account_settings_crypto_prefer_encrypt">Encryption preference</string>
+    <string name="account_settings_crypto_prefer_encrypt_on">By default when replying to encrypted, or if all recipients also checked this option</string>
```

Comment:
`also checked this option` -> `also would like encryption`

Reply:
This is a really tough one. Everyone generally "likes" or "prefers" encryption over not having that. But it's quite specifically checking this particular option (and only in an Autocrypt-capable client) that triggers it... hmm. I wonder if we should have a specific name (that can be exposed to users) for this setting it in the spec.